test/integration/request-body/raw.test.js

```diff
@@ -41,7 +41,7 @@ describe('Request Body Mode: raw', function () {
                 responseBody = JSON.parse(response.stream.toString());
 
             expect(response).to.have.property('code', 200);
-            expect(responseBody).to.have.property('data', 'POSTMAN');
+            expect(responseBody.data).to.be.eql({});
```

Comment:
Why there's no `data`? Is `Content-Type` required for postman-echo?

Reply:
Yes, there is no data when we remove content-type.